.gitpod.yml

```diff
@@ -0,0 +1,7 @@
+tasks:
+  - init: yarn install
+    command: >
+      yarn build &&
+      ./marp-cli.js demo/demo.md &&
+      cd demo &&
+      gp open demo.html
```

Comment:
I just not try but it might be good to use server mode instead of showing specified Markdown. Extra HTML won't be generated.

```yaml
ports:
  - port: 8080
    onOpen: open-preview # or open-browser
tasks:
  - init: yarn install
    command: yarn build && ./marp-cli.js -s demo
```

Reply:
I did not know the server mode, it is useful when I tried.